src/tests/ftest/util/dfuse_utils.py

```diff
@@ -41,6 +43,7 @@ def __init__(self, namespace, command):
         self.sys_name = FormattedParameter("--sys-name {}")
         self.singlethreaded = FormattedParameter("--singlethreaded", False)
         self.foreground = FormattedParameter("--foreground", False)
+        self.log = getLogger(__name__)
```

Comment:
I can remove this, however having this here will create a new logger object that logs with the correct __name__ won't it?  If it inherits one from ObjectWithParemeters then it would get the __name__ from there.

Reply:
You are correct.  The issue is with https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/command_utils.py#L150.  If this is changed to:
```
self.log = getLogger(self.__class__.__name__)
```
It will work as we intend without needing to repeat `self.log = getLogger(__name__)` in `dfuse_utils.py`:
```
>>> import logging
>>> logging.basicConfig(format="%(asctime)s - %(name)s - %(levelname)s: %(message)s", datefmt="%H:%M:%S", level=logging.DEBUG)

>>> import command_utils
>>> a = command_utils.ObjectWithParameters("foo")
>>> a.log.info("This is a")
16:15:37 - ObjectWithParameters - INFO: This is a

>>> import dfuse_utils
>>> d = dfuse_utils.DfuseCommand("foo", "bar")
>>> d.log.info("This is d")
16:16:09 - DfuseCommand - INFO: This is d
```
If you are up for it I'd also recommend making the same change on https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/command_utils.py#L50.

Reply:
Will do.  This will use the class name rather than the module (file) name but I think that's probably better anyway.